ansible_collections/arista/avd/roles/eos_designs/doc/fabric-variables.md

```diff
@@ -21,6 +23,9 @@
 **Variables and Options:**
 
 ```yaml
+# Shutdown peer interfaces of a device that is not deployed.
```

Comment:
same comments as above for the variable name and the description

Reply:
should be clearer now